sample_factory/envs/doom/doom_gym.py

```diff
@@ -463,8 +473,13 @@ def close(self):
         except RuntimeError as exc:
             log.warning("Runtime error in VizDoom game close(): %r", exc)
 
-        if self.viewer is not None:
-            self.viewer.close()
+        # if self.viewer is not None:
+        #     self.viewer.close()
```

Comment:
Unless it serves a purpose, try not to leave commented code like this. To find old code we can always use git :)